health/health.go

```diff
@@ -161,16 +177,31 @@ type warnOptFunc func(*Warnable)
 
 func (f warnOptFunc) mod(w *Warnable) { f(w) }
 
-// Warnable is a health check item that may or may not be in a bad warning state.
-// The caller of NewWarnable is responsible for calling Tracker.SetWarnable to update the state.
+// WarnableCode is a string that is used to distinguish each Warnable from others.
+type WarnableCode string
```

Comment:
// It is globally unique within the program.

Reply:
done